tests_v2/test_focus_resize.py

```
@@ -0,0 +1,318 @@
+    wait_for,
+    wait_for_surface_command_roundtrip,
+    workspace_panes,
+    focused_pane_id,
+)
+
+
+BUNDLE_ID = os.environ.get("CMUX_BUNDLE_ID", "com.cmuxterm.app.debug")
+DEFAULT_SOCKET_PATHS = [
+    os.environ.get("CMUX_SOCKET", ""),
+    os.path.expanduser("~/Library/Application Support/cmux/socket"),
+    "/tmp/cmux-debug.sock",
```

Ruff (0.15.9)

[error] Probable insecure usage of temporary file or directory: "/tmp/cmux-debug.sock"

(S108)